scripts/bundleSize/bundleSizeConfig.js

```diff
@@ -1,6 +1,6 @@
 module.exports = {
   // Size limit for all bundles used by each service (K)
   // Keep these +/- 5K and update frequently!
-  MIN_SIZE: 837,
-  MAX_SIZE: 899,
+  MIN_SIZE: 686,
+  MAX_SIZE: 833,
```

Comment:
Are we still following the rule of "Keep these +/- 5K and update frequently" mentioned in the comment on line 2-3 above?

![image](https://user-images.githubusercontent.com/30599794/89283986-f897b200-d645-11ea-9b6f-db0157038973.png)
And ran the script locally and noticed that these min/max sizes are not following this rule.

Reply:
With the changes, the smallest page may be significantly smaller than the largest; I have always read that statement as meaing the individual values should not change significantly by +/- 5k. Should we change that comment to make it clearer?

Reply:
I read it as we have to add 5kB to the largest and minus 5kB from the smallest bundle. This way when people make small changes it will not have to be updated as frequently. 

So I guess we could update the comment to be something like "Keep the MAX_SIZE +5 above largest value and MIN_SIZE -5 below smallest value". Something on the lines of that, just so it's explicit

Reply:
Updated to match your suggestions @thekp, I think it is more explicit b43131e 👍

Reply:
Thanks for the update Andrew, just one more small suggestion. I think the current min/max values also need an update to follow our rule of +/- 5.

Reply:
I thought it was more that 'over time' they shouldn't shift more than 5 +/- in either direction 😕 rather than being maintained +/- 5 below the current values? Anyone else got any thoughts, just wanna get this comment right, we still aren't quite reading the same thing from the comment 🙈

Reply:
I thought if someone was to make a small change that affects our bundle. It would increase/decrease our bundle by a little bit so the reason we have it +/- 5 for the max and min is that developers don't have to frequently update it on every change to the codebase. 👀

Reply:
I've always understood MIN/MAX_SIZE values as allowing for minor deviation in bundle sizes before alerting - otherwise alerts would be constant. With this in mind, MAX_SIZE would be maintained as (maximum bundle size plus our allowed deviation of 5kB) and anything over this threshold would create an alert. The same for MIN_SIZE respectively.

Reply:
Thanks for the second opinion @simonsinclair I'll update the description a bit to make that super clear, thanks for sticking with this too @thekp  🙂

Reply:
Applied our agreed understanding with a new comments and threshods 5k either side of the limits: https://github.com/bbc/simorgh/pull/7229/commits/9dcb55322768ae9a5e602b3098fecf0129751939